external/protoc/CMakeLists.txt

```diff
@@ -21,17 +21,13 @@ function(emil_fetch_protocol_buffer_compiler)
     FetchContent_MakeAvailable(protoc)
 
     if (CMAKE_HOST_WIN32)
-        set(EMIL_PROTOC_COMPILER_BINARY "${protoc_SOURCE_DIR}/bin/protoc.exe" CACHE INTERNAL "")
-    else()
-        set(EMIL_PROTOC_COMPILER_BINARY "${protoc_SOURCE_DIR}/bin/protoc" CACHE INTERNAL "")
+        set(host_executable_postfix ".exe")
```

Comment:
Can CMAKE_EXECUTABLE_SUFFIX be used instead of this?

Reply:
No, because that is for the target not the host. So if you cross-compile it can be set to .elf.

Reply:
Right, of course!